tsdb/postings_for_matchers_cache.go

```diff
@@ -31,14 +37,15 @@ type IndexPostingsReader interface {
 // NewPostingsForMatchersCache creates a new PostingsForMatchersCache.
 // If `ttl` is 0, then it only deduplicates in-flight requests.
 // If `force` is true, then all requests go through cache, regardless of the `concurrent` param provided.
```

Comment:
nit: had to look up what's `concurrent`, maybe
```suggestion
// If `force` is true, then all requests go through cache, regardless of the `concurrent` param provided to the PostingsForMatchers method.
```